<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtFunctionPlotterPlot2dDialog</class>
 <widget class="QDialog" name="QtFunctionPlotterPlot2dDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FunctionPlotter</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="widget_equation" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>y(x)=</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_equation">
        <property name="text">
         <string>cos(x)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_main" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QWidget" name="widget_bottom" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="2">
          <widget class="QDoubleSpinBox" name="box_maxx"/>
         </item>
         <item row="1" column="0">
          <widget class="QDoubleSpinBox" name="box_minx">
           <property name="minimum">
            <double>-999999999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>999999999.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_maxx">
           <property name="text">
            <string>x_max</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="spacer_bottom">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_minx">
           <property name="text">
            <string>x_min</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QScrollArea" name="area_plot">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="plot_contents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>558</width>
           <height>389</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
